packages/devextreme/js/ui/popup/ui.popup.js

```diff
@@ -491,15 +497,17 @@ const Popup = Overlay.inherit({
 
             item.toolbar = data.toolbar || item.toolbar || 'top';
 
-            if(item && item.toolbar === toolbar) {
+            if(item?.toolbar === toolbar) {
                 if(isShortcut) {
                     extend(item, { location: data.location }, this._getToolbarItemByAlias(data));
-                }
-
-                const isLTROrder = currentPlatform === 'generic';
-
-                if((data.shortcut === 'done' && isLTROrder) || (data.shortcut === 'cancel' && !isLTROrder)) {
-                    toolbarsItems.unshift(item);
+                    if(data.shortcut === 'done' || data.shortcut === 'cancel') {
```

Comment:
You can use ACTION_BUTTONS_ORDER const here for not using a simple string:

`ACTION_BUTTONS_ORDER.includes(data.shortcut)`

Aggregate simple strings to arrays or objects looks better to me. But it is minor problem.